_template_python/README.md

```diff
@@ -24,34 +24,42 @@ the following:
 env/bin/python -m delphi_NAME
 ```
 
-Once you are finished with the code, you can deactivate the virtual environment
-and (optionally) remove the environment itself.
+If you want to enter the virtual environment in your shell, 
+you can run `source env/bin/activate`. Run `deactivate` to leave the virtual environment. 
+
+Once you are finished, you can remove the virtual environment and 
+params file with the following:
 
 ```
-deactivate
-rm -r env
+make clean
 ```
 
 ## Testing the code
 
-To do a static test of the code style, it is recommended to run **pylint** on
-the module. To do this, run the following from the main module directory:
+To run static tests of the code style, run the following command:
 
 ```
-env/bin/pylint delphi_NAME
+make lint
 ```
 
-The most aggressive checks are turned off; only relatively important issues
-should be raised and they should be manually checked (or better, fixed).
 
 Unit tests are also included in the module. To execute these, run the following
 command from this directory:
 
 ```
-(cd tests && ../env/bin/pytest --cov=delphi_NAME --cov-report=term-missing)
+make test
 ```
 
+To run individual tests, run the following:
+
```

Comment:
Move the command to occur directly after the colon here, and put the clarification afterwards.

Reply:
fixed